tests/phpunit/tests/interactivity-api/wpInteractivityAPIFunctions.php

```diff
@@ -417,4 +417,45 @@ public function test_process_directives_in_tags_that_dont_visit_closer_tag() {
 		unregister_block_type( 'test/custom-directive-block' );
 		$this->assertEquals( '1', $processor->get_attribute( 'src' ) );
 	}
+
+	/**
+	 * Tests that context from void tags is not propagated to next tags.
+	 *
+	 * @ticket 60768
+	 *
+	 * @covers wp_interactivity_process_directives_of_interactive_blocks
+	 */
+	public function test_process_context_directive_in_void_tags() {
+		register_block_type(
+			'test/custom-directive-block',
+			array(
+				'render_callback' => function () {
+					return '<div data-wp-interactive="nameSpace" data-wp-context=\'{"text": "outer"}\'><input id="first-input" data-wp-context=\'{"text": "inner"}\' data-wp-bind--value="context.text" /><input id="second-input" data-wp-bind--value="context.text" /></div>';
```

Comment:
Should there be another test for the `data-wp-interactive` directive that also runs logic in the exiting mode?

Reply:
Added in [this commit](https://github.com/WordPress/wordpress-develop/pull/6267/commits/86bf0fbc590cb96ebedcd1e772ef8befca82e620).